style_editor.html

```diff
@@ -8,14 +8,167 @@
   <title>ProffieOS Style Editor</title>
 <style>
 
-.button_on {
-  background: lightblue;
+/* Make page elements non-selectable with drag box */
+/* Took a vote, enabled won, but disabled for now. */
+/** {
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}*/
+
+/* Box sizing helps keep buttons uniform in
+size when padding and margins may change */
+* {
+  box-sizing: border-box;
+}
+
+/* Submit, copy, etc... */
+.middle-btn-container {
+  white-space: nowrap;
```

Comment:
nowrap seems like it would not work well if the screen is narrow

Reply:
This was also just moving inlined styling to CSS. 
I can't see it making a difference.
There aren't enough buttons to span the width of the smallest browser window I can size, nor any issue/difference on mobile.
Want it to go away?

Reply:
yes, let's take it away
it's likely that we'll have more buttons in the future

Reply:
Done.